src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -37,6 +37,7 @@ sealed partial class App : Application
 		public App()
 		{
 			ConfigureFilters(LogExtensionPoint.AmbientLoggerFactory);
+			ConfigureFeatureFlags();
```

Comment:
Why it's not called from a static constructor?  Is it because you don't want to enlist it until it's actually used?

Reply:
No particular reason...